tracer/src/Datadog.Trace/Sampling/AgentSamplingRule.cs

```diff
@@ -143,8 +148,7 @@ public SampleRateKey(string service, string env)
 
             public static SampleRateKey? Parse(string key)
             {
-                // Expected format:
-                // service:{service},env:{env}
+                // Expected format: "service:{service},env:{env}"
```

Comment:
nit: we have helpers for parsing this sort of logic more efficiently using Span now IIRC.

I wouldn't have mentioned it, but you touched the function, so now you're it 😛

Reply:
Yeah, I almost made changes here yesterday to avoid the `String.Split()` calls. This make me realize we don't have any unit tests for `SampleRateKey`, so maybe I'll do both of those things in a separate PR.